modules/fts/pages/fts-supported-queries.adoc

```diff
@@ -1,4 +1,5 @@
 = Supported Queries
+:page-aliases: query-types.adoc 
 
 [abstract]
 With Full Text Search you can perform queries on Full Text Indexes. You can perform the Queries either by using Couchbase Web Console, the Couchbase REST API, N1QL (using search functions in the Query service), or the Couchbase SDK.
```

Comment:
You can perform the Queries -> You can perform the queries

Reply:
👍

Reply:
Updated